lib/cc/engine/analyzers/python/main.rb

```diff
@@ -12,11 +12,20 @@ module Python
         class Main < CC::Engine::Analyzers::Base
           LANGUAGE = "python"
           DEFAULT_PATHS = ["**/*.py"]
-          DEFAULT_MASS_THRESHOLD = 40
-          BASE_POINTS = 1000
+          DEFAULT_MASS_THRESHOLD = 32
+          BASE_POINTS = 1_500_000
+          POINTS_PER_OVERAGE = 50_000
+
+          def calculate_points(mass)
```

Comment:
I like lifting to a base class as well! However, since we're updating the tuning algorithm language-by-language, I was keeping separate. 

Some of the constants might be different per language as well.

I lean toward favoring moving into a base class as a nice separate refactor PR after, once changes are in place.